pkg/handler/templates.go

```diff
@@ -293,6 +296,46 @@ func (th *TemplateHandler) deleteTemplate(c echo.Context) error {
 	return c.NoContent(http.StatusNoContent)
 }
 
+// GetTemplateRepoConfigurationFiles godoc
+// @Summary      Get configuration files for all repositories in a template
+// @ID           getTemplateRepoConfigurationFiles
+// @Tags         repositories
+// @Accept       json
+// @Produce      text/plain
+// @Param        template_uuid  path  string    true  "Identifier of the template"
+// @Success      200   {string} string
+// @Failure      400 {object} ce.ErrorResponse
+// @Failure      401 {object} ce.ErrorResponse
+// @Failure      404 {object} ce.ErrorResponse
+// @Failure      500 {object} ce.ErrorResponse
+// @Router       /templates/{template_uuid}/config.repo [get]
+func (th *TemplateHandler) getTemplateRepoConfigurationFiles(c echo.Context) error {
+	_, orgID := getAccountIdOrgId(c)
+	templateUUID := c.Param("template_uuid")
+
+	template, err := th.DaoRegistry.Template.Fetch(c.Request().Context(), orgID, templateUUID, false)
+	if err != nil {
+		if errors.Is(err, gorm.ErrRecordNotFound) {
+			err = &ce.DaoError{NotFound: true, Message: "Could not find template with UUID " + templateUUID}
+		}
+		return ce.NewErrorResponse(ce.HttpCodeForDaoError(err), "Error fetching template", err.Error())
+	}
+
+	xRHID := identity.GetIdentity(c.Request().Context())
+	if xRHID.Identity.Type == "System" {
+		if template.OrgID != xRHID.Identity.OrgID && template.OrgID != xRHID.Identity.Internal.OrgID {
+			return ce.NewErrorResponse(http.StatusForbidden, "Error authorizing client", "System OrgID does not match template OrgID")
+		}
+	}
```

Comment:
Is this additional check still needed? It doesn't seem like system sets the orgID any differently than user

Reply:
it might not be? i think the real test will be in stage once we can test with a client. i'll remove this for now and we can just add it back in if we find it's necessary

Reply:
ah okay, makes sense. we'll test it and see.